README.md

```diff
@@ -632,115 +683,29 @@ module "mssql_database" {
-| users.password                         | string       | Password for the user login                                                                                                                                                                              | yes      |                                  |
-| users.roles                            | list(string) | List of roles to attach to the user                                                                                                                                                                      | yes      |                                  |
-| collation                              | string       | The collation of the database                                                                                                                                                                            | no       | `"SQL_Latin1_General_CP1_CI_AS"` |
-| sku_name                               | string       | The sku for the database. For vCores, this also sets the maximum capacity                                                                                                                                | no       | `"GP_S_Gen5_1"`                  |
-| max_size_gb                            | number       | The maximum size of the database in gigabytes                                                                                                                                                            | no       | `16`                             |
-| min_capacity                           | number       | The minimum vCore of the database. The maximum is set by the sku tier. Only relevant when using a serverless vCore based database. Set this to 0 otherwise.                                              | no       | `0.5`                            |
-| storage_account_type                   | string       | The storage account type used to store backups for this database. Possible values are Geo, Local and Zone                                                                                                | no       | `"Local"`                        |
-| auto_pause_delay_in_minutes            | number       | Time in minutes after which database is automatically paused. A value of -1 means that automatic pause is disabled. Only relevant when using a serverless vCore based database. Set this to 0 otherwise. | no       | `60`                             |
-| tags                                   | map(string)  | Map of tags to assign to the resources                                                                                                                                                                   | no       | `{}`                             |
-
-##### Outputs
-
-| Output                     | Type   | Description                                                |
-|----------------------------| ------ |------------------------------------------------------------|
```

Comment:
I don't think this should have been removed 🙃

Reply:
🙃 I have no idea how this happened, but I fixed it 👍🏼